modules/panorama/variables.tf

```diff
@@ -149,3 +149,14 @@ variable "metadata" {
   type        = map(string)
   default     = {}
 }
+
+variable "service_account" {
```

Comment:
We stick to the pattern: 
```
variable "example" {
   description = ""
   type        = ""
   default     = ""
 }
```
:)

Reply:
Re-arranged the order and also added description !